app/src/main/java/space/taran/arkretouch/Preferences.kt

```diff
@@ -0,0 +1,38 @@
+package space.taran.arkretouch
+
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import space.taran.arkretouch.di.DIManager
+import space.taran.arkretouch.presentation.drawing.EditManager
+import java.nio.file.Files
+import javax.inject.Inject
+import kotlin.text.Charsets.UTF_8
+
+class Preferences @Inject constructor() {
+
+    private val appContext = DIManager.component.app()
+
+    suspend fun persistDefaults(editManager: EditManager) {
```

Comment:
Actually, here JSON could be better. In arklib (and Navigator) we use custom storage format because of the 2 reasons:
1) plain text format should be faster to parse;
2) custom text format has only 1 possible representation, this is better for synchronization.

Custom text format has its own downsides, like need to support custom format. For app properties we don't need these points 1 and 2, so just using standard JSON format and serialization should be better.